src/layer/park.js

```diff
@@ -3,8 +3,15 @@
 import * as Label from "../constants/label.js";
 import * as Color from "../constants/color.js";
 
+const parkLayerFilter = [
+  "in",
+  ["get", "class"],
+  ["literal", ["nature_reserve", "national_park", "protected_area"]],
```

Comment:
[`class`](https://openmaptiles.org/schema/#park) can potentially be anything, based on `protection_title` taggified. Do we need to include other common values like [`protected_site`](https://openmaptiles.org/schema/#park) and [`přírodní_památka_(pp)`](https://taginfo.openstreetmap.org/tags/protection_title=P%C5%99%C3%ADrodn%C3%AD%20pam%C3%A1tka%20(PP)#overview)?

Reply:
Oh, I thought I fixed that but I see that I never finished openmaptiles/openmaptiles#1491, so I'll change that to exclusion

Reply:
Changed in 9642b23.
I also see now that aboriginal_lands is not differentiated at the zoom so I've also removed park layer rendering from z4 until we can get a fix.